### ISSUE TITLE
Naming conflict with ZString

### ISSUE THREAD

**Issue by shirok1** [Cysharp/UniTask#401]
```text
Library\PackageCache\com.cysharp.unitask@2.3.1\Runtime\UnityBindingExtensions.cs(15,80): error CS0118: 'Text' is a namespace but is used like a type
```

Can simply be resolved by replacing `Text` to fully specified `UnityEngine.UI.Text`.

**Comment by neuecc** [comment]
I see, fully specified is no side-effect, so let's change it.

**Comment by neuecc** [comment]
fixed at `2.3.3`.
